go.mod

```diff
@@ -119,44 +112,54 @@ require (
 	go.opencensus.io v0.22.2 // indirect
 	gocloud.dev v0.9.0
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
 	golang.org/x/tools v0.0.0-20200415034506-5d8e1897c761
 	gopkg.in/AlecAivazis/survey.v1 v1.8.3
-	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.0.0-20190816222004-e3a6b8045b0b
-	k8s.io/apiextensions-apiserver v0.0.0-20190718185103-d1ef975d28ce
-	k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
+	istio.io/api v0.0.0-20191115173247-e1a1952e5b81
+	istio.io/client-go v0.0.0-20191120150049-26c62a04cdbc
+	k8s.io/api v0.17.2
+	k8s.io/apiextensions-apiserver v0.0.0-20191114105449-027877536833
+	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/helm v2.7.2+incompatible
-	k8s.io/kube-openapi v0.0.0-20190228160746-b3a7cee44a30
-	k8s.io/kubernetes v1.11.3
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
+	k8s.io/kubernetes v1.14.0
 	k8s.io/metrics v0.0.0-20180620010437-b11cf31b380b
 	k8s.io/test-infra v0.0.0-20190131093439-a22cef183a8f
-	knative.dev/pkg v0.0.0-20191217184203-cf220a867b3d
+	knative.dev/pkg v0.0.0-20200207181514-32ea84581573
+	knative.dev/serving v0.12.1-0.20200210194206-365600fcbe27
 	sigs.k8s.io/yaml v1.1.0
 
 )
 
-replace k8s.io/api => k8s.io/api v0.0.0-20190528110122-9ad12a4af326
+replace k8s.io/api => k8s.io/api v0.16.5
```

Comment:
Have you tried moving some of these replace statements back into the dependency list? It would be nice to get rid of them, but I guess we can experiment with this once the bulk of the move is in.

Reply:
So something somewhere in the dependency tree keeps insisting on replacing the normal dependency list version with 1.17.2, so gotta keep the replace for now, sadly.

Reply:
> So something somewhere in the dependency tree keeps insisting on replacing the normal dependency list version with 1.17.2, so gotta keep the replace for now, sadly.

Fair enough. Still a huge step forward. I think one of the reasons I failed with my attempts was that I at the same time tried to get rid of the replace statements and it always kept failing one way or another.